content/applications/services/fsm/default_warehouse.rst

```diff
@@ -3,10 +3,11 @@ User default warehouse
 ======================
 
 Setting up a **default warehouse** can be useful for field technicians who keep a supply in their
-van or those who always resupply from the same warehouse.
+van or those who always resupply from the same warehouse. It also allows field workers to switch
+between warehouses from their profile.
```

Comment:
It seems this is the main point you want to add in this PR, yet, you don't explain how to do it. Please add an explanation, as the current one is only accessible as an admin, and not as a regular user.

Reply:
Indeed, thank you! I added it as a tip (.. tip::
   To switch your own default warehouse from anywhere in the database, click your **profile
   icon** in the upper right corner of the screen. Then, go to :menuselection:`My Profile -->
   Preferences --> Default Warehouse`.) Let me know if it makes sense to you.

Reply:
Updated to two headings as discussed